scripts/requirements-base.txt

```
@@ -18,6 +18,7 @@ canopen
 packaging
 progress
 psutil
+py7zr
```

If this is windows only, maybe?

```suggestion
py7zr; sys_platform == "win32"
```

---

I applied your suggestion.